Libs/MRML/Core/vtkMRMLScene.cxx

```diff
@@ -1137,7 +1137,7 @@ vtkMRMLNode*  vtkMRMLScene::AddNodeNoNotify(vtkMRMLNode *n)
       // their references.
       std::string newId(sn->GetID());
       std::string oldId(n->GetID() ? n->GetID() : sn->GetID());
-      if (oldId != newId)
+      if (oldId != newId && !oldId.empty())
```

Comment:
Oh this is the other instance, not the line that actually caused the problem, I get it now! Yes, let's check if it's NULL